src/main/java/com/statful/client/framework/springboot/registry/StatfulRegistryAutoConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package com.statful.client.framework.springboot.registry;
+
+import com.statful.client.domain.api.StatfulClient;
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.Resource;
+
+@Configuration
+@EnableConfigurationProperties(value = StatfulMetricsProperties.class)
+public class StatfulRegistryAutoConfiguration {
```

Comment:
This should be in a different package (different maven module IMO, see other comment)